zrml/prediction-markets/src/tests.rs

```diff
@@ -360,7 +360,11 @@ fn create_categorical_market_deposits_the_correct_event() {
         let market_id = 0;
         let market = MarketCommons::market(&market_id).unwrap();
         let market_account = PredictionMarkets::market_account(market_id);
-        assert!(event_exists(Event::MarketCreated(0, market_account, market)));
+        System::assert_last_event(Event::PredictionMarkets(crate::Event::MarketCreated(
+            0,
+            market_account,
+            market,
+        )));
```

Comment:
```rust
fn check_last(event: PEvent<Runtime>) {
    System::assert_last_event(Event::PredictionMarkets(event));
}
```

```suggestion
       
check_last(PEvent::MarketCreated(
            0,
            market_account,
            market,
        ));
```

Reply:
The problem here is that we may want to check for events emitted from other pallets. But I've managed to improve the aesthetics a bit.